src/main.rs

```diff
@@ -1,7 +1,17 @@
+use std::{env, process};
 use code_visualizer::renderer;
 
 fn main() {
-    let paths = renderer::get_unicode_files_in_dir("./input/");
+    let mut code_dir = env::args()
+        .nth(1)
+        .unwrap_or_else(|| {
```

Comment:
Instead of forcing the user to specify a directory, why not just use the current default `./input/` if no arg is specified?
i.e.
```rust
// snip
let mut code_dir = env::args()
    .nth(1)
    .unwrap_or("./input/");
```

Reply:
This was something I suggested as an alternative, though honestly I think we should just deprecate the old method